src/plot.js

```diff
@@ -69,12 +69,12 @@ export function plot(options = {}) {
       if (facetData == null) throw new Error("missing facet data");
       facetChannels = {};
       if (x != null) {
-        const fx = Channel(facetData, {value: x, scale: "fx"});
+        const fx = Channel(facetData, {value: x, scale: "fx", transform: options.fx?.transform});
```

Comment:
We should use applyScaleTransforms to do this. This isn’t doing exactly the same thing (e.g., the percent option).